src/components/Player/Information/Information.js

```diff
@@ -6,6 +6,7 @@ import Text from './Text';
 
 class Information extends Component {
     render() {
+
       return (
```

Comment:
Friendly Tip:

It's good to get into the habit at looking at what you're committing, before you do add files to your commit - and try to avoid committing files with just a single whitespace change. If a file needs to be reformatted, it's best to do it in its own commit, or to reformat a bunch of files in one shot. Otherwise, it just adds noise to your PR, and makes it harder to read. (And it may give the impression that you didn't look at what you were doing, before committing.)

Reply:
How can I check the content of the commit? I usually add all changed files...I could add individually but not sure how to check what was changed. I believe I tried something in this component and then deleted it leaving the extra space. I want to prevent that from creating clutter because I will definitely be trying/deleting a lot.